apps/rig_api/lib/rig_api/router.ex

```diff
@@ -37,4 +37,23 @@ defmodule RigApi.Router do
     pipe_through(:api)
     get("/", HealthController, :check_health)
   end
+
+  scope "/swagger-ui" do
+    forward("/", PhoenixSwagger.Plug.SwaggerUI,
+      otp_app: :rig_api,
+      swagger_file: "rig_api_swagger.json"
+    )
+  end
+
+  def swagger_info do
+    Application.ensure_all_started(:rig_api, :temporary) 
+    %{
+      info: %{
+        version: Application.spec(:rig_api,:vsn) |> to_string,
```

Comment:
whitespace (comma)... just as a comment, I've never seen `to_string` explicitly used anywhere; I think the idiomatic way to do this would be

```elixir
"#{Application.spec(:rig_api, :vsn)}"
```